README.Rmd

```diff
@@ -72,6 +72,23 @@ You can install the development version from [GitHub](https://github.com/) with:
 devtools::install_github("trytoolchest/toolchest-client-r")
 library(toolchest)
 ```
+
+### Python requirements
+The R client requires a version of **Python 3.6 or greater** with 
+`libpython` libraries, accessible via the `$PATH`. 
+
+Most manually installed versions of Python will have the necessary libraries
+installed with Python itself.
```

Comment:
Do automatically installed versions of Python on Windows and our test Linux have the installed libraries?

Reply:
This seems to work with Linux on the jupyter notebooks. I'll test with a new instance before merging this in.